src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,13 +2,18 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
+    @Inject
     private ProductParser productParser;
+    @Inject
     private FileReaderService fileReaderService;
```

Comment:
Consider using more informative variable names for your injected dependencies. For example, instead of 'productParser', you could use 'productParserService' to make it clear that it's a service. Similarly, 'fileReaderService' could be renamed to reflect what kind of files it reads if it's specialized, like 'productFileReaderService'.